### ISSUE TITLE
Update Documentation to Include Keyboard Shortcuts

### ISSUE THREAD

**Issue by erinmikailstaples** [heartexlabs/label-studio#3900]
**Is your feature request related to a problem? Please describe.**
_From community member Kallen Williams_
Keyboard Shortcuts are always evolving and changing, and having a page in our docs about the various keyboard shortcuts would be super valuable to users as they are getting started with their product. 

> "could you include a section on your documentation outlining the keyboard shortcuts?  Assuming they'd be slightly different for the various browsers but it'd help show the range of features available.  For example, i'm supposing there's a shortcut to edit the original imported data, but have no idea what that might be or find any verification.. I think i may have done it by accident when learning the tool but don't know what caused it to use it purposefully"

**Describe the solution you'd like**
A singular page in our Open Source and Enterprise docs about the Keyboard Shortcuts.  Bonus points for instructions on how to edit them. 

**Describe alternatives you've considered**
Currently, the users have to scroll through endless pages to get the keyboard shortcuts.... and that's a mess.

**Comment by KallenXXX** [comment]
yes, i'm definitely interested to hear more about this.  I tend to use shortcuts often to access features instinctively rather than searching and clicking multiple levels.  thanks for posting this up.

**Comment by makseq** [comment]
@KallenXXX Are you aware of this page in label studio app? 
![image](https://user-images.githubusercontent.com/501780/228994818-e45d8cb3-d425-41d8-bc85-54bfb10f71f8.png)

**Comment by AbubakarSaad** [comment]
Hello,

There has been lot of updates in the documentation. Please let us know if there is anything missing.
